CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ will publish the fix and acknowledge your finding on our site if you so wish.
 
 ## Proposed tasks to get started
 
-There is a `helpwanted` tag on some tasks in the Mender issue tracker
+There is a `helpwanted`, and a `Part` tag on some tasks in the Mender issue tracker
```

Comment:
I don't think we should document the `Part` tag, which is specifically for part timers. This is not the same thing as it may include tasks that are easy, but which require special access, for example. That's why we have two. 🙂

Reply:
Ohhh... TIL